olive/evaluator/olive_evaluator.py

```diff
@@ -415,81 +413,83 @@ def get_inference_settings(cls, metric: Metric, model: ONNXModelHandler) -> Dict
         if model_infrerence_settings:
             inference_settings.update(model_infrerence_settings)
 
-        metric_inference_settings = metric.get_inference_settings(cls.framework.lower())
+        metric_inference_settings = metric.get_inference_settings(Framework.ONNX.lower())
         if metric_inference_settings:
             inference_settings.update(metric_inference_settings)
 
         return inference_settings
 
-    def _inference(
-        self,
+
+class OnnxEvaluator(OliveEvaluator, OnnxEvaluatorMixin, framework=Framework.ONNX):
+    def __init__(self):
+        super().__init__()
+
+    @staticmethod
+    def get_session_wrapper(
         model: ONNXModelHandler,
         metric: Metric,
         dataloader: Dataset,
```

Comment:
Dataloader instead Dataset?

Reply:
The original type hint was `Dataset`. I think that is right since we are expecting a `torch.utils.Dataset` here.

Reply:
Actually you are right, it is supposed to be `torch.utils.data.Dataloader` so the original type hints for this file was incorrect. I will open a separate PR to fix it first.

Reply:
https://github.com/microsoft/Olive/pull/969